R/bump-version.R

```diff
@@ -33,10 +33,12 @@ bump_version_impl <- function(which) {
 get_main_branch <- function() {
   remote <- "origin"
   if (remote %in% gert::git_remote_list()$name) {
-    get_main_branch_remote(remote)
-  } else {
-    get_main_branch_config()
+    remote_main <- get_main_branch_remote(remote)
+    if (length(remote_main)) return(remote_main)
```

Comment:
I've seen that on GHA the remote main branch couldn't be found (when the remote is one we created). I think it's easier to fall back on the "local" main branch rather than trying to tweak the local remote set up, given all of this is only for demonstration.

Reply:
mmmh actually something is very wrong with the remote on GHA https://61a49ee1a1ca8a006fbc9856--angry-boyd-339aeb.netlify.app/dev/articles/demo.html#create-initial-news-md-file-1, I'll fix